.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -42,11 +42,12 @@ jobs:
 
   run-updater:
     needs:
-      - build-updater
+    - build-updater
     runs-on: ubuntu-latest
     env:
       STACKROX_NVD_API_KEY: ${{ secrets.NVD_API_KEY }}
       STACKROX_NVD_API_CALL_INTERVAL: 6s
+      MANUAL_URL: "https://raw.githubusercontent.com/stackrox/stackrox/14d2bbaa7d6b45ad4b22544ed063d20f1f73f2f8/scanner/updater/manual/vulns.json"
```

Comment:
It would be nice to use `https://raw.githubusercontent.com/stackrox/stackrox/${{ github.event.pull_request.head.sha }}/scanner/updater/manual/vulns.json` if this was triggered by a pull request. We would not have to do this when testing changes to this workflow.

Actually, maybe there is a way to reference the master SHA when scheduled. Anyway, fine to follow up later.